packages/sync-actions/src/types-actions.js

```diff
@@ -152,7 +152,7 @@ export function actionsMapFieldDefinitions(
         if (diffValue.length === 3 && diffValue[2] === 3) {
           actions.push({
             action: 'changeFieldDefinitionOrder',
-            fieldNames: next,
+            fieldNames: next.map((n) => n.name),
```

Comment:
doc: https://docs.commercetools.com/api/projects/types#change-the-order-of-fielddefinitions